neural_compressor/evaluation/lm_eval/models/huggingface.py

```diff
@@ -116,11 +116,14 @@ def __init__(
         peft: Optional[str] = None,
         autogptq: Optional[Union[bool, str]] = False,
         pad_to_buckets: Optional[Union[bool]] = False,
+        buckets: Optional[list] = [32, 64, 128, 256, 512, 1024, 2048, 4096],
```

Comment:
why is 8192 removed?  result holding of too much memory?

Reply:
yes, and it's not used. 4096 is used by wikitext